packages/celotool/src/cmds/account/invite.ts

```diff
@@ -12,39 +14,102 @@ export const describe = 'command for sending an invite code to a phone number'
 
 interface InviteArgv extends AccountArgv {
   phone: string
-  fast: boolean
 }
 
 export const builder = (yargs: Argv) => {
-  return yargs
-    .option('phone', {
-      type: 'string',
-      description: 'Phone number to send invite code,',
-      demand: 'Please specify phone number to send invite code',
-    })
-    .option('fast', {
-      type: 'boolean',
-      default: false,
-      description: "Don't download artifacts, use this for repeated invocations",
-      demand: 'Please specify phone number to send invite code',
-    })
+  return yargs.option('phone', {
+    type: 'string',
+    description: 'Phone number to send invite code,',
+    demand: 'Please specify phone number to send invite code',
+  })
+}
+
+export async function convertToContractDecimals(
+  value: number | BigNumber,
+  contract: StableTokenWrapper
+) {
+  const decimals = new BigNumber(await contract.decimals())
+  const one = new BigNumber(10).pow(decimals.toNumber())
+  return one.times(value)
 }
 
 export const handler = async (argv: InviteArgv) => {
-  console.log(`Sending invitation code to ${argv.phone}`)
+  const phone = argv.phone
+
+  console.log(`Sending invitation code to ${phone}`)
+  await execCmd('gcloud config set project celo-testnet')
+  await execCmd(
+    'gcloud kms decrypt --ciphertext-file=twilio-config.enc --plaintext-file=twilio-config.js \
+    --key=github-key --keyring=celo-keyring --location=global'
+  )
   const cb = async () => {
-    await execCmd(
-      `yarn --cwd ../protocol run invite -n ${argv.celoEnv} -p ${argv.phone} -f ${argv.fast}`
+    const kit = newKit('http://localhost:8545')
+    const account = (await kit.web3.eth.getAccounts())[0]
+    console.log(`Using account: ${account}`)
+    kit.defaultAccount = account
+
+    // TODO(asa): This number was made up
+    const verificationGasAmount = new BigNumber(10000000)
+    // TODO: this default gas price might not be accurate
+    const gasPrice = 100000000000
+
+    const temporaryWalletAccount = await kit.web3.eth.accounts.create()
+    const temporaryAddress = temporaryWalletAccount.address
+    // Buffer.from doesn't expect a 0x for hex input
+    const privateKeyHex = temporaryWalletAccount.privateKey.substring(2)
+    const inviteCode = Buffer.from(privateKeyHex, 'hex').toString('base64')
+
+    const [goldToken, stableToken, attestations, escrow] = await Promise.all([
+      kit.contracts.getGoldToken(),
+      kit.contracts.getStableToken(),
+      kit.contracts.getAttestations(),
+      kit.contracts.getEscrow(),
+    ])
+    const verificationFee = new BigNumber(
+      await attestations.attestationRequestFees(stableToken.address)
+    )
+    const goldAmount = verificationGasAmount.times(gasPrice).toString()
```

Comment:
We don't need to send gold anymore at all, do we?

Reply:
I'm not sure if anyone uses this + pays gas in gold so I just added a comment to remove it - just hesitant to do it when not everyone will have a chance to look at this